### PR TITLE
Update Roboto Flex description and add/fix profiles

### PR DESCRIPTION
**Pull request by davelab6**
This only changes catalog content, not the font files

### CODE REVIEW

File: catalog/designers/ilyaruderman/info.pb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+designer: "Ilya Ruderman"
+link: ""
+avatar {
+  file_name: "ilyaruderman.png"
```
Comment (RosaWagner) [inline catalog/designers/ilyaruderman/info.pb:4]: png -> jpeg

File: catalog/designers/irenevlachou/info.pb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+designer: "Irene Vlachou"
```
Comment (RosaWagner) [inline catalog/designers/irenevlachou/info.pb:1]: Irene is not credited in any GF fonts, in which family should she be added

File: catalog/designers/mikhailstrukov/info.pb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+designer: "Mikhail Strukov"
+link: ""
+avatar {
+  file_name: "mikhailstrukov.png"
```
Comment (RosaWagner) [inline catalog/designers/mikhailstrukov/info.pb:4]: png -> jpeg

File: catalog/designers/mikhailstrukov/bio.html [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<p>
+Mikhail Strukov is a type designer and graduate of Ilya Ruderman’s course at BHSAD (Moscow) and Frank Blokland's program at Plantin Institute of Typography (Antwerp), where he developed a deep interest in history of type and evolution of type production.
+He collaborates with CSTM Fonts and Samarskaya & Partners, working mostly on Cyrillic versions for both custom and retail typefaces.
+When not busy with designing Cyrillic, he reviews, consults and writes about Cyrillic.
+A smalltown resident, road cycling fan, and builder of local communities
+</p>
```
Comment (RosaWagner) [inline catalog/designers/mikhailstrukov/bio.html:6]: missing a webpage link

File: catalog/designers/irenevlachou/info.pb (hunk 2 above)
Comment (RosaWagner) [reply catalog/designers/irenevlachou/info.pb:1]: Ah my bad, I see the change to metadata.pb in another file

**Review by m4rc1e — COMMENTED** [review]
Can designer profile pics be .jpg?

These are the only jpgs we have:

```
Marcs-Air-2:fonts marcfoley$ find catalog/designers/ -iname "*.jpg"
catalog/designers//flaviazimbardi/flavia_zimbardi.jpg
catalog/designers//yuryostromentsky/yuryostromentsky.jpg
catalog/designers//mikhailstrukov/mikhailstrukov.jpg
```

File: catalog/designers/mikhailstrukov/bio.html (hunk 4 above)
Comment (davelab6) [reply catalog/designers/mikhailstrukov/bio.html:6]: None was supplied, its oK

File: ofl/robotoflex/DESCRIPTION.en_us.html [hunk 5]
```diff
@@ -1,6 +1,31 @@
...
+With Flex, you can customize Roboto to express and finesse your text in ways never before possible.
+Today, people are constantly switching between devices, resizing browsers, and spreading our viewports across multiple screens.
+So Google commissioned Font Bureau to re-imagine Roboto to “flex” along with us, with a special emphasis on large-screen capabilities.
+This was achieved by amplifying the original design to an extreme range of weights, grades, widths and optical sizes.
 </p>
 <p>
-    To contribute, see <a href="https://github.com/googlefonts/roboto-flex">github.com/googlefonts/roboto-flex</a>.
-</p>
\ No newline at end of file
+The second benefit of Roboto Flex is the designer’s ability to fitness and fine-tune their designs with <a href="https://fonts.google.com/knowledge/glossary/parametric_axes">parametric axes</a>.
```
[the remark this answers is not in the document]
Comment (davelab6) [reply ofl/robotoflex/DESCRIPTION.en_us.html:9]: Yes

**Comment by davelab6** [comment]
> > The metadata.pb looks like an older version, since then the languages stuff have be removed and the axes values has changed a bit, I wonder why the diff is not showing it.
> 
> @RosaWagner imo it looks ok. I see Nate ripped out the language stuff several days ago [https://github.com/google/fonts/blob/main/ofl/robotoflex/METADATA.pb](https://github.com/google/fonts/blob/main/ofl/robotoflex/METADATA.pb?rgh-link-date=2022-04-27T15%3A54%3A17Z)

The diff is clean so I think its OK to merge. I'll do it

**Comment by davelab6** [comment]
Updated [to_sandbox](https://github.com/google/fonts/commit/09c84fcd88aea453197ace0ef03bc66455ef8175 ) too

**Comment by davelab6** [comment]
@RosaWagner 

> and the axes values has changed a bit

Hmm, actually I'm not sure what you are referring to here? :)

**Comment by RosaWagner** [comment]
Did you guys looked at the diff that I attached ????

**Comment by RosaWagner** [comment]
I confused, it's only PNG that are allowed, all jpeg files must be turned to png otherwise Rod is unhappy

**Pull request merged**